### PR TITLE
(chore): Explicit declaration of source(bucket name) of suggested manifests

### PR DESCRIPTION
**Pull request by HUMORCE**
Relates to https://github.com/ScoopInstaller/Scoop/issues/5131

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/docker-nightly.json [hunk 1]
```diff
@@ -31,17 +31,14 @@
...
-        "Docker Machine": "docker-machine",
-        "Kubernetes Standalone": "minikube",
-        "Kubernetes Client": "kubectl",
-        "compose.yml to kubernetes.yml": "kompose"
+        "sudo": "gsudo",
+        "Docker Compose": "main/docker-compose",
+        "Docker Machine": "main/docker-machine",
+        "Kubernetes Client": "main/kubectl",
+        "Kubernetes Standalone": "main/minikube",
+        "Linux container support on Windows": "unknown/lcow",
```
Comment (HUMORCE) [inline bucket/docker-nightly.json:39]: This manifest appears to have been lost. https://github.com/ScoopInstaller/Main/pull/2972

Comment (rashil2000) [reply bucket/docker-nightly.json:39]: I remember putting it somewhere in the `deprecated` folder...

Comment (HUMORCE) [reply bucket/docker-nightly.json:39]: After being removed from main bucket, it was not created anywhere.

https://github.com/search?q=org%3AScoopInstaller+lcow&type=commits

**Comment by HUMORCE** [comment]
https://github.com/ScoopInstaller/Versions/commit/1f6e09255aa99984bcc67492a5565fc9f194dd4c, https://github.com/ScoopInstaller/Versions/commit/d6a12037002a597016f5fee1ffa8ca216c45fd86, https://github.com/ScoopInstaller/Versions/commit/9b10ddaf15ab65af581f74b003b63558d98121e6, https://github.com/ScoopInstaller/Versions/commit/4ab86c901ca72910332835048d33c9b4f10bfdd1, https://github.com/ScoopInstaller/Versions/commit/8b108ae6502968900d75270dd9e6a0d4c6efd978, https://github.com/ScoopInstaller/Versions/commit/6e26f21c3be6cadc97c93672c655a2befeaa0c5c, https://github.com/ScoopInstaller/Versions/commit/fcb48931c6aac35bf3d7af3de5d56e88c90ad41b, https://github.com/ScoopInstaller/Versions/commit/3a4291c8de1fe1809e9545ea65d93aa9d6daffe8, https://github.com/ScoopInstaller/Versions/commit/4de0a3cbac621ce6efced5214696840d6fe24c3c, https://github.com/ScoopInstaller/Versions/commit/144e00398272bd67f61b36848be32ce0b91c23d6, https://github.com/ScoopInstaller/Versions/commit/0cda6bd0e57085cc0a80a478f2555e2930e9ac3c, https://github.com/ScoopInstaller/Versions/commit/f5eb74be20fa0ccd8734dc51ff55bb1fcf667cff, https://github.com/ScoopInstaller/Versions/commit/f882d7b624211bb2245de64f7167a794ee419ff1, https://github.com/ScoopInstaller/Versions/commit/36c51681da7602b3237e52f961dfe27ada08ee19, https://github.com/ScoopInstaller/Versions/commit/c9ccf8c6965773c483020143ab7fc87464d445ec, https://github.com/ScoopInstaller/Versions/commit/7442f07a579c0e9d486a2a1a6a0446c5449dbe78, https://github.com/ScoopInstaller/Versions/commit/ec96d0f6b15a039ba7a49351e0ad37ee7f78dc00, https://github.com/ScoopInstaller/Versions/commit/ec96d0f6b15a039ba7a49351e0ad37ee7f78dc00, https://github.com/ScoopInstaller/Versions/commit/c5c02bb504809f4e76499c404eee6d1daf95d14f

**Pull request closed without merging**